common/src/main/scala/net/psforever/packet/game/ChatMsg.scala

```diff
@@ -0,0 +1,102 @@
+// Copyright (c) 2016 PSForever.net to present
+package net.psforever.packet.game
+
+import net.psforever.packet.{GamePacketOpcode, Marshallable, PacketHelpers, PlanetSideGamePacket}
+import scodec.Codec
+import scodec.codecs._
+
+// ChatMessageTypes were reversed by:
+  // Checking the type in the packet upon receipt by server while using slash commands
+  // Replaying the ChatMsg packet back to the sender after modifying the messagetype value
+
+// Some other unchecked commands, if they actually exist as message types:
+// Reply (/r), BattleGroup (/bg), ReplyCSR (/gmreply)
+
+object ChatMessageType extends Enumeration {
```

Comment:
Would it possible to this ChatMessageType to separate file to keep packet files somewhat clean?
Could you create a new package `net.psforever.types` and place it as `ChatMessageType`? I want to do this eventually as having all game types separate from packets will help readability down the road